### ISSUE TITLE
-log option not processed?

### ISSUE THREAD

**Issue by paulstodghill** [compomics/peptide-shaker#205]
I tried to change the directory into which logs are written using the "-log" option. This results in a error message that -log option isn't recognized.

Here's what the -help message says,

```
Optional Log Folder:

-log                                 Folder where to write logs.

```
Here's what happens,

```
daffy$ java -Djava.awt.headless=true -Xmx4g -cp /storage/ps27/work/projects/proteomic-pipeline/bin/../lib/PeptideShaker/PeptideShaker-1.13.3.jar eu.isas.peptideshaker.cmd.PeptideShakerCLI -log .
<CompomicsError>PeptideShaker processing failed. See the PeptideShaker log for details.</CompomicsError>
org.apache.commons.cli.UnrecognizedOptionException: Unrecognized option: -log
        at org.apache.commons.cli.Parser.processOption(Parser.java:363)
        at org.apache.commons.cli.Parser.parse(Parser.java:199)
        at org.apache.commons.cli.Parser.parse(Parser.java:85)
        at eu.isas.peptideshaker.cmd.PeptideShakerCLI.main(PeptideShakerCLI.java:928)
daffy$ 
```

Thanks!

**Comment by hbarsnes** [comment]
Hi Paul,

Thanks for telling us about this. There was a minor error in our code. We'll release a new version next week that should solve the problem. However, even when the log parameter is properly recognized your command line above will not run as it's missing all the mandatory parameters for the [PeptideShakerCLI](
https://github.com/compomics/peptide-shaker/wiki/PeptideShakerCLI#a---peptideshakercli) command line.

If you only want to set the log location (or any of the other paths) please use [PathSettingsCLI](https://github.com/compomics/peptide-shaker/wiki/PeptideShakerCLI#e---pathsettingscli) instead.

Also, note that relative paths may cause problems, so I'd recommend using complete paths if possible. 

Best regards,
Harald

**Comment by paulstodghill** [comment]
Thanks for the quick response.

**Comment by hbarsnes** [comment]
Hi Paul,

We've just released PeptideShaker v1.13.4 which should fix this issue. If this is not the case please let us know and we'll reopen the issue.

Best regards,
Harald
